xlators/mgmt/glusterd/src/glusterd-syncop.c

```diff
@@ -2590,60 +2587,45 @@ gd_sync_task_begin(dict_t *op_ctx, rpcsvc_request_t *req)
         goto out;
     }
 
-    /* Based on the op_version, acquire a cluster or mgmt_v3 lock */
-    if (cluster_lock) {
```

Comment:
The local variable should also be removed

Reply:
will update in the next patch.

Reply:
done